hoot-services/src/main/java/hoot/services/controllers/ingest/ImportCommand.java

```diff
@@ -86,6 +86,11 @@
             //options.add("schema.translation.script=" + translationPath);
         //}
 
+        if (!isNoneTranslation && (classification == SHP) || (classification == FGDB) || (classification == ZIP)) 
```

Comment:
Looking ahead, should this include GeoPackage as well?
Or, compare "classification" with a config option list of formats.

Reply:
I had just copied and pasted that line from another spot in the file...hadn't too closely before. It looks like those are tied directly to an enumeration, so seems ok. Moving them to a config file would only buy you functionality if you had some kind of dynamic registration for the file formats, which I don't think services has...I may not completely understand the services import code though.

Reply:
I hadn't thought about services. I was thinking about making it easier for the Hoot team to add support for different output formats.

Reply:
Ok. Yeah, we pretty much have that in core. You just have to write the reader class and correctly implement the isSupported method to pick up the format and `hoot convert` or `hoot conflate` takes care of all the rest of it. If its a streaming format and you implemented the partial reading interfaces, they take care of that too.

Reply:
Ahhhhhhhh. This is Java. Now I understand your comment. My bad.

Probably still should add GeoPackage as an option.